terraform/modules/quorum-cluster-region/quorum-cluster-region-backup-event.tf

```diff
@@ -0,0 +1,399 @@
+        "Service": "lambda.amazonaws.com",
+        "Service": "events.amazonaws.com",
+        "Service": "sns.amazonaws.com"
+      },
+      "Effect": "Allow",
+      "Sid": ""
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_iam_policy" "BackupLamba_Permissions" {
+  count = "${signum(lookup(var.maker_node_counts, var.aws_region, 0))}"
+  path = "/"
+  description = "IAM policy for accesing EC2 and S3 buckets from Lambda"
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": [
+        "ec2:*"
+      ],
+      "Effect": "Allow",
+      "Resource": "*"
+    },
+    {
+      "Action": ["s3:*"],
+      "Effect": "Allow",
+      "Resource": [
+        "${aws_s3_bucket.quorum_backup.arn}",
+        "${aws_s3_bucket.quorum_backup.arn}/*"
+      ]
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_iam_role_policy_attachment" "BackupLambda1" {
+   count = "${signum(lookup(var.maker_node_counts, var.aws_region, 0))}"
+   role = "${aws_iam_role.iam_for_BackupLambda.name}"
+   policy_arn = "${aws_iam_policy.BackupLamba_Permissions.arn}"
+}
+
+
+resource "aws_iam_policy" "BackupLambda_logging" {
+  count = "${signum(lookup(var.maker_node_counts, var.aws_region, 0))}"
+  name = "BackupLambda-${var.network_id}-${var.aws_region}"
+  path = "/"
+  description = "IAM policy for logging from a lambda"
+
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": [
+        "logs:*"
+      ],
+      "Resource": "arn:aws:logs:*:*:*",
+      "Effect": "Allow"
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_iam_role_policy_attachment" "BackupLambda2" {
+   count = "${signum(lookup(var.maker_node_counts, var.aws_region, 0))}"
+   role = "${aws_iam_role.iam_for_BackupLambda.name}"
+   policy_arn = "${aws_iam_policy.BackupLambda_logging.arn}"
+}
+
+# Zip the Backup Lambda
+resource "null_resource" "BackupLambda" {
+  count = "${signum(lookup(var.maker_node_counts, var.aws_region, 0))}"
+  triggers {
+    main = "${sha256(file("${var.BackupLambda_source_file}"))}"
+  }
+  # destroy
+  provisioner "local-exec" {
+    when = "destroy"
+    command = "rm ${var.aws_region}-${var.BackupLambda_output_path}"
+    on_failure = "continue"
+  }
+  // # creat
+  // provisioner "local-exec" {
+  //   command = "zip -j ${var.aws_region}-${var.BackupLambda_output_path} ${var.BackupLambda_source_file}"
+  // }
+  # the creation is in "archive_file" "BackupLambda"
+}
+
+data "archive_file" "BackupLambda" {
+  count = "${signum(lookup(var.maker_node_counts, var.aws_region, 0))}"
+  type        = "zip"
+  source_file  = "${var.BackupLambda_source_file}"
+  output_path = "${var.aws_region}-${var.BackupLambda_output_path}"
+  depends_on = ["null_resource.BackupLambda"]
+}
+
+resource "aws_kms_key" "a" {
+  count = "${signum(lookup(var.maker_node_counts, var.aws_region, 0))}"
+  description = "Used for encrypting SSH keys on S3"
+  tags {
+     name = "BackupLambda-${var.network_id}-${var.aws_region}-KMS"
+  }
+}
+
+output "aws_kms_key_test" {
```

Comment:
Outputs should not be mixed with the rest of the configuration. Also, this looks like maybe it was left around when it shouldn't be?